modules/nw-creating-project-and-service.adoc

```diff
@@ -6,23 +6,21 @@
 [id="nw-creating-project-and-service_{context}"]
 = Creating a project and service
 
-If the project and service that you want to expose do not exist, first create
-the project, then the service.
+If the project and service that you want to expose do not exist, create the project and then create the service.
 
-If the project and service already exist, skip to the procedure on exposing the
-service to create a route.
+If the project and service already exist, skip to the procedure on exposing the service to create a route.
 
 .Prerequisites
 
-* Install the `oc` CLI and log in as a cluster administrator.
+* Install the {oc-first}}and log in as a cluster administrator.
```

Comment:
I'm not sure what I'm looking at here.  "`oc` CLI" is perfectly understandable to me, but what will it get replace with? I'm sure this is a docs things, just want to understand

Reply:
Oh, this is a docs thing. We have an attribute for this. I'm just ensuring I align with some docs expectations :).

![Screenshot from 2024-08-29 11-05-38](https://github.com/user-attachments/assets/9ae7cdfd-7df7-4b84-b791-a3113540af91)